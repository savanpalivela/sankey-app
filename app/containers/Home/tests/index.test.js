/**
 *
 * Tests for Home
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

// import React from 'react';
// import { render } from 'react-testing-library';
// import { IntlProvider } from 'react-intl';
// import {
//   INCOME_5,
//   EXPENSE_5,
//   NODE_5,
//   LINK_1,
// } from '../../../../internals/mocks/data';
// import 'jest-dom/extend-expect'; // add some helpful assertions

// import { Home } from '../index';
// import { DEFAULT_LOCALE } from '../../../i18n';

describe('<Home />', () => {
  it('Expect to not log errors in console', () => {});
});
