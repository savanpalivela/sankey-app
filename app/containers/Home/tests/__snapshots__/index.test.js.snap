// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render and match the snapshot 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Home
    </title>
    <meta
      content="Description of Home"
      name="description"
    />
  </HelmetWrapper>
  <h3>
    <FormattedMessage
      defaultMessage="Cashflow"
      id="app.containers.Home.header"
      values={Object {}}
    />
  </h3>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    >
      <Record
        attributeKeyList={
          Array [
            "services",
            "recurring",
            "project",
            "consultation",
            "interest",
          ]
        }
        handleAddAttRecord={[Function]}
        handleRemoveAttRecord={[Function]}
        handleValueChange={[Function]}
        title="income"
      />
    </div>
    <div
      className="col-sm-6"
    >
      <Record
        attributeKeyList={
          Array [
            "operations",
            "nonOperating",
            "fixed",
            "variable",
            "tax",
          ]
        }
        handleAddAttRecord={[Function]}
        handleRemoveAttRecord={[Function]}
        handleValueChange={[Function]}
        title="expense"
      />
    </div>
  </div>
</div>
`;
